<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleCameraSettingsWidget</class>
 <widget class="QWidget" name="SingleCameraSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QnGeneralCameraSettingsWidget" name="generalCameraSettingsWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="recordingTab">
      <attribute name="title">
       <string>Recording</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QnCameraScheduleWidget" name="cameraScheduleWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="motionTab">
      <attribute name="title">
       <string>Motion</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1,0,0,0">
       <item>
        <widget class="QLabel" name="motionAvailableLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Motion is not available for this camera</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="motionWidget" native="true"/>
       </item>
       <item>
        <widget class="QGroupBox" name="motionSettingsGroupBox">
         <property name="title">
          <string>Motion settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Motion detection:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QRadioButton" name="cameraMotionButton">
              <property name="text">
               <string>Hardware (Camera built-in)</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="softwareMotionButton">
              <property name="text">
               <string>Software (Max FPS may be slightly decreased on Server)</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Sensitivity:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QnTextTickSlider" name="sensitivitySlider">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum">
             <number>9</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
            <property name="tickInterval">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Motion mask (0)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Maximum sensitivity (9)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Camera motion web page:</string>
            </property>
            <property name="openExternalLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="motionWebPageLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>URL</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="resetMotionRegionsButton">
           <property name="text">
            <string>Reset motion regions</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fisheyeTab">
      <attribute name="title">
       <string>Dewarping</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QnFisheyeSettingsWidget" name="fisheyeSettingsWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="expertTab">
      <attribute name="title">
       <string>Expert</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QnAdvancedSettingsWidget" name="expertSettingsWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnCameraScheduleWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/properties/camera_schedule_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnAdvancedSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/properties/expert_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnTextTickSlider</class>
   <extends>QSlider</extends>
   <header>ui/widgets/properties/text_tick_slider.h</header>
  </customwidget>
  <customwidget>
   <class>QnFisheyeSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/properties/fisheye_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnGeneralCameraSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/properties/general_camera_settings_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>buttonClicked(QAbstractButton*)</slot>
 </slots>
</ui>
